rotkehlchen/tests/api/test_uniswap.py

```diff
@@ -116,7 +116,7 @@ def test_get_balances(
         for lp_asset in lp['assets']:
             lp_asset_type = type(lp_asset['asset'])
 
-            assert lp_asset_type in (str, dict)
+            assert lp_asset_type in {str, dict}
```

Comment:
Actually this is also wrong. I tested and str/dict are not considered constants for the interpreter. will undo